### PR TITLE
feat: add fab button #142

### LINKED ISSUE #118: Addd a more intuitve way to upload files
The users should have a more "easy to find" upload button.

A big + button on the left can be a good solution.
Another option is to highlight the current upload button or to move it somewhere else.

### PR DESCRIPTION
**Pull request by nogahaskal**

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ services:
     depends_on:
       - api-gateway
   api-gateway:
-    image: 'bluehub.azurecr.io/meateam/api-gateway:develop'
+    image: 'bluehub.azurecr.io/meateam/api-gateway:merges'
```
Comment (Shahar-Y) [inline docker-compose.yml]: Keep it on develop

File: src/css/styles.css [hunk 2]
```diff
@@ -96,15 +97,14 @@
   background: var(--blue);
   color: #fff;
   border-radius: 50%;
-  font-size: .75em;
+  font-size: 0.75em;
```
Comment (Shahar-Y) [inline src/css/styles.css:100]: You really dislike the '.x' shortcut, do you?

**Review by Shahar-Y — CHANGES_REQUESTED** [review]
Looks great, just change the docker-compose.
Also, next time put the link to your working version in the pr description with specification to the branches which are not develop (other than this service's). In this case - 'merges' in api-gw.

**Comment by Shahar-Y** [comment]
Also, pay attention to the snyk comments

**Comment by Shahar-Y** [comment]
In your last commit I didn't see any code changes, except the `baseURL` deletion. was that intentional? the comment said 'add logo button'..
Also, is the latest version of this branch working in your kubeernetes environment and passes manual checks?

File: src/views/Files.vue [hunk 3]
```diff
@@ -55,7 +55,9 @@ export default {
     ...mapGetters(["selectedCount", "isListing", "isEditor", "isFiles", "isActiveDialog", "shares", "direction", "isSearch"]),
     ...mapState(["req", "user", "reload", "multiple", "loading", "path"]),
     isPreview() {
-      return !this.loading && !this.isListing && !this.isEditor;
+      let preview = !this.loading && !this.isListing && !this.isEditor;
```
Comment (Shahar-Y) [inline src/views/Files.vue]: Are you sure this is the best way to check it? Looks a bit dangerous.. What if there was another option except loading, isListing, isEditor, or preview ? We would have to remember to add it to this condition...

**Review by Shahar-Y — CHANGES_REQUESTED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (nogahaskal) [reply docker-compose.yml]: done

File: src/views/Files.vue (hunk 3 above)
Comment (nogahaskal) [reply src/views/Files.vue]: change that and use state instead.

File: src/views/Files.vue [hunk 4]
```diff
@@ -52,7 +52,7 @@ export default {
     Preview
   },
   computed: {
-    ...mapGetters(["selectedCount", "isListing", "isEditor", "isFiles", "isActiveDialog", "shares", "direction", "isSearch"]),
+    ...mapGetters(["isPreivew", "selectedCount", "isListing", "isEditor", "isFiles", "isActiveDialog", "shares", "direction", "isSearch"]),
```
Comment (Shahar-Y) [inline src/views/Files.vue]: I think you misspelled `preview `

Comment (Shahar-Y) [reply src/views/Files.vue]: Check where else it might have happened and update the Kubernetes

**Review by Shahar-Y — CHANGES_REQUESTED** [review]

File: src/views/Files.vue (hunk 4 above)
Comment (nogahaskal) [reply src/views/Files.vue]: it wasn't used so i deleted that

**Review by Shahar-Y — APPROVED** [review]

**Pull request merged**